### PR TITLE
[pt_BR]: updated translation for PT_BR

### PR DESCRIPTION
**Pull request by KayckMatias**
Updated some translations for PT_BR that were missing.

### CODE REVIEW

File: locales/pt_BR/json.json [hunk 1]
```diff
@@ -906,7 +906,7 @@
     "This subscription cannot be resumed. Please create a new subscription.": "Esta assinatura não pode ser retomada. Por favor, crie uma nova assinatura.",
     "This subscription has expired and cannot be resumed. Please create a new subscription.": "Esta assinatura expirou e não pode ser retomada. Por favor, crie uma nova assinatura.",
     "This team has too many team members for the selected plan.": "This team has too many team members for the selected plan.",
-    "This team is not eligible for this plan.": "This team is not eligible for this plan.",
+    "This team is not eligible for this plan.": "Esta equipa não é elegível para este plano.",
```
Comment (EuCarlos) [inline locales/pt_BR/json.json:909]: The translation of team into Brazilian Portuguese is incorrect. We can use `Equipe` or `Time`.

File: locales/pt_BR/json.json [hunk 2]
```diff
@@ -995,7 +995,7 @@
     "View": "Visualizar",
     "View Invitations": "Ver convites",
     "View Receipt": "Veja o recibo",
-    "Virgin Islands": "Virgin Islands",
+    "Virgin Islands": "Ilhas Virgins",
```
Comment (EuCarlos) [inline locales/pt_BR/json.json:998]: In Brazilian Portuguese, the correct is `Ilhas Virgens` not `Ilhas Virgins`

**Review by EuCarlos — CHANGES_REQUESTED** [review]
Only this is incorrect, when fixed this will be ready for merge.

File: locales/pt_BR/json.json (hunk 2 above)
Comment (KayckMatias) [reply locales/pt_BR/json.json]: sorry, it went unnoticed

**Comment by KayckMatias** [comment]
@EuCarlos I updated the ones you mentioned and finished translating what was missing into Brazilian Portuguese, check if everything is right :)

**Pull request merged**